presto-main/src/main/java/com/facebook/presto/sessionpropertyproviders/NativeWorkerSessionPropertyProvider.java

```diff
@@ -90,6 +91,11 @@ public NativeWorkerSessionPropertyProvider(FeaturesConfig featuresConfig)
                         "Native Execution only. Reduce() function will throw an error if it encounters an array of size greater than this value.",
                         100000,
                         !nativeExecution),
+                integerProperty(
+                        NATIVE_EXPRESSION_MAX_COMPILED_REGEXES,
+                        "Native Execution only. Controls maximum number of compiled regular expression patterns per regular expression function instance per thread of execution.",
```

Comment:
nit: this line is too long. Is there lint warning? If not feel free to ignore this comment.

Reply:
No in java code.
I checked - there are some long lines and even longer than this.
I can split it to multiple rows.